pyneuroml/swc/ExportSWC.py

```diff
@@ -124,6 +124,9 @@ def convert_to_swc(nml_file_name, add_comments=False, target_dir=None):
     lines = []
     comment_lines = []
 
+    if not nml_doc.cells:
```

Comment:
From what I remember, `nml_doc` will always have `cells`, but it can be an empty list---so perhaps we need to check the length of this here instead.

Reply:
okay, I updated the check to check len of `nml_doc.cells` instead